metal-detector-firmware/src/uavcan/uavcan_node.cpp

```diff
@@ -0,0 +1,136 @@
+#include <ch.h>
+#include <hal.h>
+#include <chprintf.h>
+
+#include <uavcan/uavcan.hpp>
+#include <uavcan_stm32/uavcan_stm32.hpp>
+#include <uavcan/protocol/SoftwareVersion.hpp>
+
+#include <version/version.h>
+
+#include <main.h>
+#include "uavcan_node.h"
+#include "parameter_server.hpp"
+
+#include <cvra/metal_detector/EMIRawSignal.hpp>
```

Comment:
Compilation fails here:
```
Compiling uavcan_node.cpp
./src/uavcan/uavcan_node.cpp:15:10: fatal error: cvra/metal_detector/EMIRawSignal.hpp: No such file or directory
 #include <cvra/metal_detector/EMIRawSignal.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```
Did you forget to add the custom UAVCAN datatypes?

Reply:
Good catch. Yes I forgot to add the file while rebasing 😅 It's fixed now